src/components/views/FilesDropdown.tsx

```diff
@@ -83,6 +87,63 @@ const isFileValidForFigure = (
   }
 }
 
+const getFigures = (doc: ProsemirrorNode) => {
```

Comment:
if possible please move these functions out of the tsx files into lib folder

Reply:
moved